src/Host/Client/Impl/Session/RSession.cs

```diff
@@ -571,8 +573,9 @@ internal sealed class RSession : IRSession, IRCallbacks {
         /// <param name="url"></param>
         /// <returns></returns>
         Task IRCallbacks.WebBrowser(string url) {
```

Comment:
Thinking a bit more about it. If we define a broker-rhost URL conversion protocol. something like:
```
[http://localhost:1212/myhelp.html](http://localhost:1212/myhelp.html) <-> [http://brokerurl:2020/localurl/1212/myhelp.html](http://brokerurl:2020/localurl/1212/myhelp.html)
```
RHost can make this conversion when open_url is called, and then broker can handle it as a normal web request.
In this case, we won't have to make any changes in VS part at all

---
In reply to: [77244150](https://github.com/Microsoft/RTVS/pull/2242#discussion_r77244150) [](ancestors = 77244150)